db/seeds.rb

```diff
@@ -6,3 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+if ENV['USE_DEMO_DATA'] || Settings.use_demo_data || Rails.env.test?
```

Comment:
The reason it is part of the MessagesController and playground is that I was asked to get it working in the playground first and then to automate it. What do you think of having it in the playground?  we will have to regularly change/test the demo data as we add new data-fields/devices and the playground is a handy place to do it.

The daily demo data job is then triggered by a sidekiq cron addon ,the advantage being to use the sidekiq GUI to see if jobs run/fail.

I can maybe move the seeds/manifests_demo_template files to app/jobs/manifests_demo_template  [prefer to keep them as individual files rather than the setup of a  task/job?